apps/src/templates/instructions/TeacherOnlyMarkdown.jsx

```diff
@@ -10,15 +10,25 @@ import color from '../../util/color';
 export default class TeacherOnlyMarkdown extends Component {
   static propTypes = {
     content: PropTypes.string,
+    hideContainer: PropTypes.bool,
   };
 
   render() {
-    const {content} = this.props;
+    const {content, hideContainer} = this.props;
 
     if (!content) {
       return null;
     }
 
+    // CodeBridge does not use the teal container/header, so we just render the content.
+    if (hideContainer) {
+      return (
+        <div style={styles.content}>
```

Comment:
Is it worth extracting these lines out into a helper method so they aren't repeated between the two return statements?

Reply:
Good idea! Done.